projects/T-beam/src/main.cpp

```diff
@@ -3,12 +3,27 @@
 #include "gps.h"
 #include "led.h"
 
+void log(void *pvParameters) {
```

Comment:
this function shouldn't be here, move it to an appropriate place

Reply:
This function will eventually be used to log data to a disk from all components. Given that the main file will naturally have to aggregate these, the placement of this function is appropriate.

Reply:
If so please add a comment saying why this function is where it is and how will it change in the future to be differently than it is now